cmd/podman/containers/logs.go

```diff
@@ -105,5 +107,10 @@ func logs(_ *cobra.Command, args []string) error {
 		logsOptions.Since = since
 	}
 	logsOptions.Writer = os.Stdout
-	return registry.ContainerEngine().ContainerLogs(registry.GetContext(), args, logsOptions.ContainerLogsOptions)
+
+	err := registry.ContainerEngine().ContainerLogs(registry.GetContext(), args, logsOptions.ContainerLogsOptions)
+	if isatty.IsTerminal(os.Stdout.Fd()) || isatty.IsCygwinTerminal(os.Stdout.Fd()) {
+		fmt.Printf("\033[0m\n")
```

Comment:
Could you define this using a variable?  `ResetAll = "\033[0m"`  or perhaps pull in https://godoc.org/github.com/whitedevops/colors and use from there?  I think defining our own constant is probably OK rather than pulling in colors.